nixos/modules/services/mail/postsrsd.nix

```diff
@@ -143,15 +252,21 @@ in
       before = [ "postfix.service" ];
       wantedBy = [ "multi-user.target" ];
       requires = [ "postsrsd-generate-secrets.service" ];
-      confinement.enable = true;
```

Comment:
I think confinement is doing more than just those hardening options and limits what files the process can read. It kinda makes debugging harder but it would be fine for me.

Reply:
Yeah, sets `RootDirectory=` and lots of `BindReadOnlyPaths=`. Not sure if that is worth the trouble.